pycbc/events/events.py

```diff
@@ -190,7 +190,8 @@ def newsnr(snr, reduced_x2, q=6., n=2.):
     ind = numpy.where(reduced_x2 > 1.)[0]
     newsnr[ind] *= ( 0.5 * (1. + reduced_x2[ind] ** (q/n)) ) ** (-1./q)
 
-    if len(newsnr) > 1:
+    # If input was floats, return a float. Otherwise return numpy array.
```

Comment:
I would prefer to say 'if inputs were floats' or 'for float inputs'

Reply:
This has been changed as suggested.